README.md

```diff
@@ -202,6 +202,6 @@ Thimble uses the [throng](https://www.npmjs.com/package/throng) module to levera
 Contact Us
 -----------
 
-We're a friendly group, so feel free to chat with us in the "Thimble" channel on Mozilla Chat running on [Mattermost](https://about.mattermost.com). To access Mozilla Chat head to [this link]( http://chat.mozillafoundation.org). Note that you will need to create an account first.
+We're a friendly group, so feel free to chat with us in the "Thimble" channel on Mozilla Chat running on [Mattermost](https://about.mattermost.com). To access Mozilla Chat head to [this link]( http://chat.mozillafoundation.org). Note that you will need to create an account first. If you have signed in already, head [here](https://chat.mozillafoundation.org/mozilla/channels/thimble) to directly visit the Thimble channel.
```

Comment:
it would be nicer if we can rephrase things here because we're pointing to a domain, and then a page on that domain (as far as URLs are concerned) so I'd suggest:
```
To access Mozilla Chat head over to [chat.mozillafoundation.org](http://chat.mozillafoundation.org). Note that you will be prompted to create an account if you do not already have one. If you already have an account, and you are already logged in from a previous visit, you can directly access the Thimble channel by clicking [this link](https://chat.mozillafoundation.org/mozilla/channels/thimble).
```